docs/PROXYSCOPES.md

```diff
@@ -0,0 +1,45 @@
+# Secure FHIR Gateway and Proxy - OAuth Scopes Guide
+## Introduction
+In order to support the SMART on FHIR authorization scheme the Secure FHIR Gateway and Proxy relies upon the OAuth capabilities provided by Azure Active Directory. This document describes how to configure the Secure FHIR Gateway and Proxy resource application registration to enable OAuth scopes.
```

Comment:
Is the statement *Secure FHIR Gateway and Proxy* referring to a single thing (ie Proxy) or two separate entities? If the latter, what do we mean by *Secure FHIR Gateway*?